tests/VolleyManagement.UnitTests/Mvc/ViewModels/DivisionViewModelEqualityComparer.cs

```diff
@@ -20,13 +20,19 @@ public static void AssertAreEqual(DivisionViewModel expected, DivisionViewModel
             Assert.AreEqual(expected.Name, actual.Name, $"{messagePrefix}[Id:{expected.Id}]Name should be equal.");
 
             Assert.AreEqual(expected.Groups.Count, actual.Groups.Count, $"[Id:{expected.Id}]Number of Groups items should be equal.");
-            for (var i = 0; i < expected.Groups.Count; i++)
+
+            var expectedGroupsEnumerator = expected.Groups.GetEnumerator();
```

Comment:
Why don't you use Assert helper you wrote earlier?

Reply:
let it be

Reply:
It does not answer my question

Reply:
?

Reply:
I must write custom Comparator for TournamentViewModel, DivisionViewModel, GroupViewModel, to use TestHelper.AreEqual and use, is it resonable